_inc/client/plans/single-product-backup.jsx

```diff
@@ -108,20 +125,24 @@ function SingleProductBackupPriceGroup( {
 			</React.Fragment>
 		);
 	}
+
+	let billingTimeFrameString = '';
+	if ( 'yearly' === billingTimeFrame ) {
+		billingTimeFrameString = __( 'per year', {
+			comment: 'Duration of product subscription timeframe.',
+		} );
+	} else if ( 'monthly' === billingTimeFrame ) {
+		billingTimeFrameString = __( 'per month', {
+			comment: 'Duration of product subscription timeframe.',
+		} );
+	}
+
 	return (
 		<div className="single-product-backup__price-group">
-			{ billingTimeFrame === 'yearly' &&
-				__( '{{price/}} {{timeframe}}per year{{/timeframe}}', {
-					components: { price, timeframe },
-					comment:
-						'Describes how much a product costs. {{price/}} can be a single value or a range of values',
-				} ) }
-			{ billingTimeFrame === 'monthly' &&
-				__( '{{price/}} {{timeframe}}per month{{/timeframe}}', {
-					components: { price, timeframe },
-					comment:
-						'Describes how much a product costs. {{price/}} can be a single value or a range of values',
-				} ) }
+			{ price }
+			<div className="single-product-backup__price-group-billing-timeframe">
+				{ billingTimeFrameString }
+			</div>
```

Comment:
@tyxla @delawski - How's this setup? I copied the  format exactly from [here](https://github.com/Automattic/jetpack/pull/14203/files#diff-7457a6cdabf24fa36923c20d4c9b101cR203-R210) to reduce translated strings, but I wasn't sure if having a string that's entirely placeholders is good for translation.